### PR TITLE
Private data fixes

### PR DESCRIPTION
**Pull request by dzikowski**

### CODE REVIEW

**Comment by dzikowski** [comment]
see: https://github.com/softwaremill/fabrica/pull/153/files#r592205127

File: .dockerignore [hunk 1]
```diff
@@ -6,3 +6,4 @@ node_modules
 Jenkinsfile
 samples/chaincodes/chaincode-java-simple
 samples/chaincodes/chaincode-kv-node/node_modules
+samples/chaincodes/chaincode-kv-node/**/*.spec.js
```
Comment (Hejwo) [inline .dockerignore:9]: :+1:

File: Jenkinsfile [hunk 2]
```diff
@@ -120,6 +120,15 @@ try {
...
 
+      stage("Test private data") {
+        try {
+          sh "e2e-network/test-03-private-data.sh"
+        } finally {
+          archiveArtifacts artifacts: 'e2e-network/test-03-private-data.sh.tmpdir/**/*', fingerprint: true
+          archiveArtifacts artifacts: 'e2e-network/test-03-private-data.sh.logs/*', fingerprint: true
+        }
+      }
+
```
Comment (Hejwo) [inline Jenkinsfile:131]: :+1:

File: docs/schema.json [hunk 3]
```diff
@@ -39,7 +39,10 @@
             "1.4.5",
             "1.4.6",
             "1.4.7",
-            "1.4.8"
+            "1.4.8",
+            "1.4.9",
+            "1.4.10",
+            "1.4.11"
```
Comment (Hejwo) [inline docs/schema.json:45]: watch out - these lines are already added to `main` branch :)

**Review by Hejwo — APPROVED** [review]
seems legit

**Pull request merged**